domains/tycho10101.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Tycho10101",
+           "email": "",
+           "discord": "914245831748632636",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.dzRWhccSL2217JXqqmZt9m_O4ll58iu-XcmyPaN-Zvqh7DnK4eebxpcMknh7bK4rhm7jdqmcFtKmmnGWaVO7n2mjvFD16Hvl0kQkQB9ZL3p2Z5mBSiOQdAgweS-iFhNSPHoAZhELYah8i4sI7x4AOZOsLoeQ5rUPdpx4raUv0bX9YyklmplyP8P__Iy13sQ0tczfj0o-Imbpeh7XnksgZfGdWMCf5Ez-ZO-hFvA60oTd9VumCAy02wCW7SJjAY8JmVy54O7yE3TdKLzMmJrQxVNZ-wdkZ4wzyAGdOHP2-dphmNDBHhOQ7asdnie7E65oaaxQUk3L43VATc4wJqLJSg.wYyNvIo2UkCm_eoYjsNgng.UsVB_ZdgSeht4eb5VZpHbT4c2-Ej8WSyFp3hMoWdPkcvTbv634WqqsYMyJ7qltpucTMPWZIzT0EL_5C22ImpLiOVCek2BBoYU-MsBKRkg1A.QSeOSDmyPG6VbNxDG0dpSg"
+        },
+    
+        "record": {
+            "CNAME": "www.tycho10101.is-a.dev.    1    in    cname tycho10101.is-a.dev."
```

Comment:
Please read our [documentation](https://is-a.dev/docs). This is a completely invalid which points at itself.